solr/core/src/test/org/apache/solr/cli/SolrCLITest.java

```diff
@@ -19,7 +19,14 @@
 import org.apache.solr.SolrTestCase;
 import org.junit.Test;
 
-public class SolrCliUptimeTest extends SolrTestCase {
+public class SolrCLITest extends SolrTestCase {
+  @Test
+  public void testResolveSolrUrl() {
+    assertEquals(SolrCLI.resolveSolrUrl("http://localhost:8983/solr"), "http://localhost:8983");
+    assertEquals(SolrCLI.resolveSolrUrl("http://localhost:8983"), "http://localhost:8983");
+    assertEquals(SolrCLI.resolveSolrUrl("http://localhost:8983/"), "http://localhost:8983");
```

Comment:
[0] Seeing these three cases makes me a little twitchy without `http://localhost:8983/solr/` to complete the set 😛

Reply:
Happy to "de-twitch" you!